s2p/geographiclib.py

```diff
@@ -80,25 +80,26 @@ def epsg_code_from_utm_zone(utm_zone):
     return const + zone_number
 
 
-def utm_proj(utm_zone):
+def crs_proj(utm_zone, crs_type="UTM"):
```

Comment:
If we drop `cfg['utm_zone']` altogether, this function could only take as input an EPSG code, and `crs_type` could be dropped.

Reply:
Removing the UTM management will be more complicated as `imscript` third-party library writes the projection comment in UTM in the .ply files.

Reply:
Could you show me where this is done?
Because when I grep for the `crs_proj` function in the project, I only find one usage here: https://github.com/lofrad/s2p/blob/4f07fc07e70d6c50998df355d7eb731bb9591f69/s2p/rasterization.py#L126
which is then written to a `tiff` using `rasterio`, but I don't see this variable being used elsewhere.

Reply:
There: https://github.com/lofrad/s2p/blob/4f07fc07e70d6c50998df355d7eb731bb9591f69/c/colormesh.c#L104

Reply:
I see. `c/colormesh.c` doesn't come from `imscript` here though, it seems that its source is directly under `c/`.

It feels a bit redundant to have two pieces of code to write a point cloud to a `ply` file (one in C in `c/colormesh.c`, and the other one in Python in `s2p/ply.py`), as it requires more maintenance cost.

Ideally, the `tri-stereo` route of the code should use `s2p/ply.py` to write its point cloud, and we could get rid of `colormesh.c` altogether?
However it seems that there is a lot going on in `colormesh`, it does more than just writing a point cloud to `ply`:
- it takes a height map as input, converts it to 3D points while potentially applying a homography and clipping points to a bounding box
- there is also a `-with-normals` option. It doesn't seem to be currently used in the project so maybe it could be dropped, but I don't know what it does exactly
- the point cloud is written to a `ply`, optionally with a color image to color the points.

I guess we will need @carlodef 's advice here to decide what to do